easybuild/easyconfigs/p/pyiron/pyiron-0.5.1-foss-2023a.eb

```diff
@@ -0,0 +1,120 @@
+easyblock = 'PythonBundle'
+
+name = 'pyiron'
+version = '0.5.1'
+
+homepage = 'https://github.com/pyiron/pyiron'
+description = "An integrated development environment (IDE) for computational materials science."
+
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('poetry', '1.5.1')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('matplotlib', '3.7.2'),
+    ('h5py', '3.9.0'),
+    ('PyYAML', '6.0'),
+    ('phonopy', '2.20.0'),
+    ('ASE', '3.22.1'),
+    ('PyTables', '3.8.0'),
+    ('nglview', '3.0.8'),
+    ('tqdm', '4.66.1'),
+    ('molmod', '1.4.8'),
+    ('scikit-learn', '1.3.1'),
+    ('yaff', '1.6.0'),
+    ('TAMkin', '1.2.6'),
+    ('QuickFF', '2.2.7'),
+    ('plotly.py', '5.16.0'),
+    ('GitPython', '3.1.40'),
+    ('Greenlet', '3.0.2'),
+    ('dill', '0.3.7'),
+    ('sympy', '1.12'),
+    ('pymatgen', '2023.12.18'),
+]
+
+check_ldshared = True
+use_pip = True
+
+exts_list = [
+    ('pint', '0.23', {
```

Comment:
We have an EC for this: `Pint 0.23` -> Move to dependencies

Reply:
Done